src/Declarations/Contracts/Generated/RemoteDependencyData.ts

```diff
@@ -0,0 +1,75 @@
+// THIS FILE WAS AUTOGENERATED
+import Domain = require('./Domain');
+'use strict';
+
+/**
+ * An instance of Remote Dependency represents an interaction of the monitored component with a remote component/service like SQL or an HTTP endpoint.
+ */
+class RemoteDependencyData extends Domain {
+  baseType: 'RemoteDependencyData' = 'RemoteDependencyData';
+
+  /**
+   * Schema version
+   */
+  public ver: number;
+
+  /**
+   * Name of the command initiated with this dependency call. Low cardinality value. Examples are stored procedure name and URL path template.
+   */
+  public name: string | undefined;
+
+  /**
+   * Identifier of a dependency call instance. Used for correlation with the request telemetry item corresponding to this dependency call.
+   */
+  public id: string | undefined;
+
+  /**
+   * Result code of a dependency call. Examples are SQL error code and HTTP status code.
+   */
+  public resultCode: string | undefined;
+
+  /**
+   * Request duration in format: DD.HH:MM:SS.MMMMMM. Must be less than 1000 days.
+   */
+  public duration: string | undefined;
+
+  /**
+   * Indication of successfull or unsuccessfull call.
+   */
+  public success: boolean;
+
+  /**
+   * Command initiated by this dependency call. Examples are SQL statement and HTTP URL's with all query parameters.
+   */
+  public data: string | undefined;
+
+  /**
+   * Target site of a dependency call. Examples are server name, host address.
+   */
+  public target: string | undefined;
+
+  /**
+   * Dependency type name. Very low cardinality value for logical grouping of dependencies and interpretation of other fields like commandName and resultCode. Examples are SQL, Azure table, and HTTP.
+   */
+  public type: string | undefined;
+
+  /**
+   * Collection of custom properties.
+   */
+  public properties: any;
```

Comment:
I believe `properties` and `measurements` are `collection` [types](https://github.com/microsoft/ApplicationInsights-Home/blob/master/EndpointSpecs/Schemas/Bond/RemoteDependencyData.bond#L47).

Reply:
Added `Properties` type to `./Domain.ts`